dGame/dGameMessages/GameMessages.h

```diff
@@ -705,7 +705,17 @@ namespace GameMessages {
 		std::vector<LDFBaseData*> localizeParams{};
 		std::u16string imageName{};
 		std::u16string text{};
-		void Send() const override;
+		void Serialize(RakNet::BitStream& bitStream) const override;
+	};
+
+	struct UseItemOnClient : public GameMsg {
+		UseItemOnClient() : GameMsg(MessageType::Game::USE_ITEM_ON_CLIENT) {}
+		LWOOBJID playerId{};
+		LWOOBJID itemToUse{};
+		uint32_t itemType{};
+		LOT itemLOT{};
+		NiPoint3 targetPosition{};
+		void Serialize(RakNet::BitStream& bitStream) const override;
```

Comment:
I like the idea of using structs for this a lot, especially if there's a way to make the process less verbose. 👍

Reply:
SendSetStun(true, true, true, true, true, true, true, true, true, true, false, true, true, true, true, false);

Reply:
can't we use . (dot) oporators with this as well?
So if we need....... the 5th bool in (EX) set to a non-deafult value, let's say `true` (idk what it is just as an example) we can just do `SetStun(.fifthvalue=true).send()` or something like that?

Reply:
You could maybe create something with a similar effect, but that syntax wouldn't work

Reply:
I.e. it might be possible to do something like
`SetStun.GetField<5>(true).send()`
but that's significantly less nice

Reply:
im fairly certain it would also have to be `SetStun{.fifthvalue=true}.send()`.  i think it would work like that as is if you're setting a couple fields, itll be tomato tomato at the end of the day as the compiler will likely optimize it away at Release

Reply:
That would only work if you were constructing a new value, right?

Reply:
i believe so, the whole thing would be in-line and would be destroyed at the end of the expression, which if you only need to change 1 field may be more clear, in some maybe not.